content/en/agent/docker/log.md

```diff
@@ -57,6 +57,17 @@ docker run -d --name datadog-agent \
            datadog/agent:latest
 ```
 
+**Note**: On Windows systems, run this command without volume mounts. That is:
```

Comment:
I believe we should just have two tabs (Linux, Windows) with the 2 commands.

Reply:
Since this is already in a tab, we can't have two tabs inside this tab.

Reply:
Oh okay, missed that.